Frameworks/UIKit/UIScrollView.mm

```diff
@@ -169,87 +162,29 @@ - (id)_dManipGestureCallback:(_UIDMPanGestureRecognizer*)gesture {
 
 - (void)_initUIScrollView {
     // Store a strongly-typed backing scrollviewer
-    _scrollViewer = rt_dynamic_cast<WXCScrollViewer>([self xamlElement]);
-    if (!_scrollViewer) {
+    WXCGrid* scrollViewGrid = rt_dynamic_cast<WXCGrid>([self xamlElement]);
+    WXCScrollViewer* scrollViewer = nil;
+
+    if (scrollViewGrid) {
+        scrollViewer = XamlControls::GetScrollViewer(scrollViewGrid);
+    } else {
+        scrollViewer = rt_dynamic_cast<WXCScrollViewer>([self xamlElement]);
+    }
```

Comment:
this should never be the case; xamlElement should never be a vanilla Xaml ScrollViewer